src/platform/user/authentication/utilities.js

```diff
@@ -284,9 +284,13 @@ export function redirect(redirectUrl, clickedEvent, type = '') {
   window.location = redirectUrl;
 }
 
-export async function mockLogin({ clickedEvent = AUTH_EVENTS.MOCK_LOGIN }) {
+export async function mockLogin(
+  { clickedEvent = AUTH_EVENTS.MOCK_LOGIN },
+  type = 'vamock',
```

Comment:
Put the `type` in the destructured object and have it default to an empty string.  Throw an error if an empty string is found.